packages/salesforcedx-vscode-core/src/util/authInfo.ts

```diff
@@ -136,7 +141,7 @@ function displayMessage(
   vsCodeWindowType?: VSCodeWindowTypeEnum,
   items?: string[]
 ) {
-  if (!isUndefined(enableWarning) && !enableWarning) {
+  if (enableWarning !== undefined && !enableWarning) {
```

Comment:
Can we remove isUndefined altogether?

Reply:
I think the method is deprecated now.

Reply:
![Screen Shot 2022-07-21 at 8 19 09 PM](https://user-images.githubusercontent.com/9795193/180335931-c66d86a3-a92e-4630-9006-61d1de089b44.png)
sure is! Looks like we have a lot of it in the code though. Mostly ConfigUtil. This looks like the right way to remove it though ^^

Reply:
Removed the refs I found 👍